BAZProjectC1/Control/ParallaxView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ParallaxView.swift
+//  BAZProjectC1g
+//
+//  Created by efloresco on 02/10/22.
+//
+
+import Foundation
+import UIKit
+
+protocol ParalaxProtocol {
+    func didSelectItem(iIndex: Int)
```

Comment:
Recuerda la [convención de nombrado de funciones en los delegates](https://github.com/raywenderlich/swift-style-guide#delegates), podrías renombrar tu método se la siguiente forma:

```suggestion
func paralaxView(_ paralaxView: ParalaxView, didSelectItem at: IndexPath)
```